### ISSUE TITLE
request to move this repo under the svelte org on github

### ISSUE THREAD

**Issue by ralyodio** [halfnelson/svelte-native#340]
I think its not getting visibility or contributions here.

**Comment by ralyodio** [comment]
the official svelte discord.

**Comment by vallemar** [comment]
@ralyodio use NativeScript discord for SvelteNative

**Comment by ralyodio** [comment]
It was dead.

**Comment by IAmRiteshKoushik** [comment]
Yeah, we need to move this repository under the SvelteOrg GitHub, and that would fix the branding issue that SvelteNative suffers from. A cross-platform alternative to React Native that can go head to head with Flutter would be really helpful. Let's talk about this the Svelte discord ?

**Comment by matthewblott** [comment]
@IAmRiteshKoushik which channel?

**Comment by maietta** [comment]
I found this on the Svelte Discord server, otherwise I would have never heard of it. Thank you for the work that's gone into this!

**Comment by IAmRiteshKoushik** [comment]
See! That is the exact problem! There is almost no visibility beyond the official discord community of Svelte! SvelteNative is not getting the love is deserves as people are unaware of this tech. It can very well become mainstream with improvements to the website, moving this repo to Svelte.org and adding in more features ?

**Comment by jonny827** [comment]
I found this by wondering myself if there was a way to use Svelte for Native Apps and Google brought me to the website and this repository as the first two links.  I definitely think it would be much easier to manage all from one organization and even possibly from one site versus two sites. People browsing the Svelte site would love to know of growing ecosystem.  Additionally, placing the repository under the main Svelte org on Github would definitely bring more attention, and a sense of longevity in the project bringing more developers to contribute to the project and bring it into mainstream use in production environments.

**Comment by jamestinman** [comment]
This really needs to happen as there is definitely a knock-on affect to Svelte adoption. A couple of CTO/tech leads I have spoken to recently say they won't consider svelte until there is an official accompanying "SvelteNative" so they can use the same skillset across devices - which means they feel compelled to stick to React / ReactNative.

Go to Svelte site and feels like an after thought, which is why - even if technically plausible rn - people are not feeling confident about adopting it in production. This is not about Rich Harris and the Svelte team jumping headlong into the codebase, just acknowledging and embracing SvelteNative more formally.

**Comment by matthewblott** [comment]
Just adding some info about Svelte Native at the bottom is insufficient in the current state. That FAQ page doesn't really look like an FAQ page. It's very disorganised and should be terser with links to explanations imo.

**Comment by halfnelson** [comment]
Good news. The new home for Svelte Native is https://github.com/nativescript-community/svelte-native and it is part of the nativescript community on npm `@nativescript-community/svelte-native`

**Comment by astremsky** [comment]
STOP FUCKING SPAMING HERE ASHOLES…. 388484 useles notifications
